clients/src/main/java/org/apache/kafka/common/errors/OffsetMetadataTooLarge.java

```diff
@@ -19,17 +19,10 @@
 
     private static final long serialVersionUID = 1L;
 
-    public OffsetMetadataTooLarge() {
```

Comment:
@granthenke I agree that it's unlikely and it's hard to be sure (especially when you consider libraries like reactive-kafka).

Sorry, I had only reviewed a few lines before I was interrupted by my little one. Sadly GitHub doesn't allow one to group notifications. I went through the rest and left a few more comments.

If you'd prefer to make the changes in bulk, you may want to wait for comments from a committer.